controllers/channels.js

```diff
@@ -143,6 +148,11 @@ module.exports = (app, passport, React, ReactDOMServer) => {
    */
   app.post('/channels/invite', async (req, res) => {
     const { data } = req.body;
+
+
+    // @TODO: req.user non-functional - get record.account from a different place
+    logger.info('\n\n\n\nInvite User\n\n\n\n', req.user);
```

Comment:
Instead of using newlines. Create a helper function that will format the text

Like

Logger.info(formatheader('Invitr U')

Reply:
we don't need this log, it was for testing, I'll remove it